packages/design-system/src/components/WIP/Accordion/Primitive/CollapsiblePanelHeader.module.scss

```diff
@@ -0,0 +1,61 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.headerWrapper {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	align-items: center;
+	gap: tokens.$coral-spacing-xxs;
+
+	background-color: tokens.$coral-color-accent-background-weak;
+	width: 100%;
+	border: none;
+	min-height: 4.4rem; // no token for this one : switch to sizing-l ?
```

Comment:
@mhuet what do you think?

Reply:
No token for the min-height because we've built it with paddings around the content. But I understand! **It works also well with the sizing-l** (I've made the test quickly) 👌🏻

Reply:
OK go for ziwin-l then 👍